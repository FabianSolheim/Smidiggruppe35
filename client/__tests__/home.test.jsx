import React from "react";
import ReactDOM from "react-dom";
import { Home } from "@mui/icons-material";

describe("Home page", () => {
  it("shows home page", () => {
    const domElement = document.createElement("div");
    ReactDOM.render(<Home />, domElement);
    expect(domElement.innerHTML).toMatchSnapshot();
  });
});
