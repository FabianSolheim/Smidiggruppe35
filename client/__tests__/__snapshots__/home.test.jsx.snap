// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page shows home page 1`] = `"<svg class=\\"MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root\\" focusable=\\"false\\" aria-hidden=\\"true\\" viewBox=\\"0 0 24 24\\" data-testid=\\"HomeIcon\\"><path d=\\"M10 20v-6h4v6h5v-8h3L12 3 2 12h3v8z\\"></path></svg>"`;
